packages/snaps-sdk/src/jsx/components/form/Button.ts

```diff
@@ -1,8 +1,14 @@
+import type { EnumToUnion } from 'src/internals';
+
 import type { SnapsChildren, StringElement } from '../../component';
 import { createSnapComponent } from '../../component';
 import type { IconElement } from '../Icon';
 import type { ImageElement } from '../Image';
 
+export enum SnapRoutes {
+  Home = 'home',
+}
+
```

Comment:
For consistency across components, could we use an union directly rather tan an enum ?